namerd/storage/consul/src/main/scala/io/buoyant/namerd/storage/consul/ConsulDtabStore.scala

```diff
@@ -19,6 +21,8 @@ class ConsulDtabStore(
 
   private[this] val log = Logger.get("consul")
 
+  def namespaceIsValid(ns: Ns): Boolean = Pattern.matches("[\\w+\\-?/?]+", ns)
```

Comment:
Also, this regex doesn't make it very clear what character set we're supporting.

Can this just be `^[A-Za-z0-9_-]+$`, for instance?